extensions/amp-pan-zoom/0.1/amp-pan-zoom.js

```diff
@@ -372,22 +372,40 @@ export class AmpPanZoom extends AMP.BaseElement {
    */
   resetContentDimensions_() {
     const content = dev().assertElement(this.content_);
-    return this.measureMutateElement(
-        () => this.measure_(),
-        () => {
-          // Set the actual dimensions of the content
-          setStyles(content, {
-            width: px(this.contentBox_.width),
-            height: px(this.contentBox_.height),
+    return this.mutateElement(() => this.clearDimensions_())
+        .then(() => {
+          return this.measureMutateElement(
+              () => this.measure_(),
+              () => this.setDimensions_(), content).then(() => {
```

Comment:
You should un-nest the `.then` here. So:

```javascript
return this.mutateElement(() => this.clearDimensions_())
  .then(() => {
    return this.measureMutateElement(
          () => this.measure_(),
          () => this.setDimensions_(), content);
  })
  .then(() => {
    const contentBox = ...;
  }).
```

Reply:
Done. Thank you!